l10n_ch_invoice_reports/models/report.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import PyPDF2
+from io import BytesIO
+from odoo import models
+
+
+class IrActionsReport(models.Model):
+    _inherit = "ir.actions.report"
+
+    def merge_pdf_in_memory(self, docs):
```

Comment:
can we use a default method?

Reply:
@i-vyshnevska  not sure what did you mean by default, afaik we use default method for gathering data for the fields. as we dont have any field here to use default, i just renamed method to be sure that it wont conflict with overriding in local-src.